guide/react/connecting.mdx

```diff
@@ -0,0 +1,38 @@
+---
+sidebar_position: 2
+---
+
+# Connecting
+
+## Prerequisites
+
+In order to connect, you need a `PARTICIPANT_TOKEN` and the `FISHJAM_URL`.
+
+## Connecting
+
+Use the `useConnect` hook to get the `connect` function.
+
+```ts
+import { connect } from "@fishjam-cloud/react-native-client";
+
+const connect = useConnect();
```

Comment:
Hmm, I think it would be better if call to hook would be embedded in dummy component (to better understand syntax)